Makefile

```diff
@@ -95,14 +95,17 @@ gen_python_sdk::
 	cd provider/cmd/${CODEGEN} && go run . python ../../../sdk/python ${SCHEMA_PATH}
 	cp ${WORKING_DIR}/README.md sdk/python
 
+build_python_sdk:: PYPI_VERSION = $(shell pulumictl convert-version -l python -v "$(VERSION)")
 build_python_sdk:: gen_python_sdk
```

Comment:
did you intend to drop the dependency on gen_python_sdk?

Reply:
Hm does this change drop a dependency? Let me try to rerun this from a clean work tree.

Reply:
sorry no, I misread the diff. Makefiles are hard

Reply:
I think the dependency is still there.. When I run `make build_python_sdk` it generates it first. This is just a Make syntax we use elsewhere for exposing env variables to the target.